===================================================================
RCS file: /cvs/gnome/nautilus-cd-burner/src/make-iso.c,v
retrieving revision 1.13
retrieving revision 1.13.2.1
diff -u -r1.13 -r1.13.2.1
--- src/make-iso.c	2006/09/29 18:23:54	1.13
+++ src/make-iso.c	2006/10/25 17:03:53	1.13.2.1
@@ -965,11 +965,11 @@
 gboolean
 nautilus_burn_iso_graft_get_info (NautilusBurnIsoGraft *graft,
 				  guint64              *size,
-				  gboolean             *use_joliet,
+				  gboolean             *use_joliet_out,
 				  GError              **error)
 {
 	GPtrArray            *argv;
-	gboolean              _use_joliet;
+	gboolean              use_joliet;
 	char                 *stdout_data;
 	char                 *stderr_data;
 	int                   exit_status;
@@ -978,7 +978,7 @@
 	char                 *filelist;
 	gboolean              result;
 
-	_use_joliet = TRUE;
+	use_joliet = TRUE;
 	argv = NULL;
 	filelist = create_temp_graft_file (graft);
 	iso_size = 0;
@@ -991,7 +991,7 @@
 
 	g_ptr_array_add (argv, "mkisofs");
 	g_ptr_array_add (argv, "-r");
-	if (_use_joliet) {
+	if (use_joliet) {
 		g_ptr_array_add (argv, "-J");
 	}
 	if (ncb_mkisofs_supports_utf8 ()) {
@@ -1062,6 +1062,9 @@
 	if (size != NULL) {
 		*size = iso_size;
 	}
+	if (use_joliet_out != NULL) {
+		*use_joliet_out = use_joliet;
+	}
 
 	return result;
 }
